ide/bugzilla/src/org/netbeans/modules/bugzilla/issue/IssuePanel.java

```diff
@@ -512,7 +508,7 @@ void reloadForm(boolean force) {
         clearHighlights();
 
         boolean isNew = issue.isNew();
-        boolean showProductCombo = isNew || !(issue.getRepository() instanceof KenaiRepository) || NBBugzillaUtils.isNbRepository(issue.getRepository().getUrl());
+        boolean showProductCombo = true;
```

Comment:
```suggestion
        boolean showProductCombo = isNew || NBBugzillaUtils.isNbRepository(issue.getRepository().getUrl());
```

or kill `NBBugzillaUtils` and change to:

```suggestion
        boolean showProductCombo = isNew;
```

Reply:
my thinking was that `!(issue.getRepository() instanceof KenaiRepository)` is essentially always evaluating to true now?

Reply:
i checked and NBBugzillaUtils is probably something for another PR (if at all possible). Since it sits in `team.commons` which has a much bigger friends list.

Reply:
@matthiasblaesing the code either displays the product property as combo box or as text field if it is a kenai project.

![new-bugzilla-issue](https://github.com/apache/netbeans/assets/114367/b61f944a-cacd-4dce-9f08-48b5a6cb985c)

with kenai gone, I believe it is ok to always display it as combo box, no?